components/blocks/cards/SignUpCard/SignUpCard.test.ts

```diff
@@ -1,7 +1,25 @@
+import createFetchMock from 'vitest-fetch-mock'
 import SignUpCard from './SignUpCard.vue'
 import { fireEvent, stubbedRender } from '@/testUtils'
+const fetchMock = createFetchMock(vi)
+fetchMock.enableMocks()
+
+afterEach(() => {
+  fetchMock.resetMocks()
+  vi.restoreAllMocks()
+})
 
 describe('<SignUpCard />', () => {
+  beforeEach(() => {
+    vi.mock('#app', () => ({
+      useRuntimeConfig: () => ({
+        public: {
+          BACKEND_BASE_URL: process.env.BACKEND_BASE_URL,
+        },
+      }),
+    }))
+  })
```

Comment:
So I think in this case, it would run 5 times in total. Once before the `it` and then before each of the other 4 `describe` blocks. That being said, I think this is overkill and we can move it to file setup instead.